src/main/java/io/prismic/Api.java

```diff
@@ -297,19 +297,33 @@ public Ref getMaster() {
   }
 
   /**
+   * Start a query defaulting on the master reference (you can still override it) with the given language
+   */
+  public Form.SearchForm query(String lang, String q) {
```

Comment:
Not sure about this. Why adding this helper?
We could call `query(q).lang('fr-fr')`.